### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by aravindhp**
- Remove ex-Red Hatters :-(
- Add Elango Sivanandam @selansen (Windows Containers team lead) :-)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,11 +1,9 @@
 approvers:
-  - abhinavdahiya
   - adambkaplan
   - aravindhp
   - ashcrow
   - bparees
   - celebdor
-  - chancez
   - cooktheryan
   - danehans
```
Comment (sdodson) [inline OWNERS:8]: @aravindhp Daneyon left too.

Comment (aravindhp) [reply OWNERS]: Removed

**Comment by sdodson** [comment]
/lgtm
I think everyone who no longer works here is removed, we can look into other strategies to make sure people are assigned who are more likely to review via other means.

**Pull request merged**